### ISSUE TITLE
Why is there a save button?

### ISSUE THREAD

**Issue by mslourens** [BoostIO/boostnote-mobile#178]
The save button only toggles the preview mode on and off, but the saving action is always excited, even when you navigate back. So why not call the save button Preview?

**Comment by RomanKlopsch** [comment]
Well, i think it would be better to set the preview as default and replace the *save*-button with an *edit*-button.
What do you think?

**Comment by mslourens** [comment]
That sounds like a sensible plan.

**Comment by openam** [comment]
I like the preview as default idea. In addition to the `edit` button can we just enter edit mode when they click on the contents of the note, like what happens in the desktop version when you are not using the split view.

**Comment by RomanKlopsch** [comment]
I think entering the edit mode by clicking on the notes content is quite annoying because it could happen accidentally. Most people use the app primary to read notes not to edit them so i would focus on the preview.

**Comment by openam** [comment]
Fair enough. I agree I'm mainly reading on my mobile. So as long as preview is default I'd be happy.
